pyanaconda/startup_utils.py

```diff
@@ -347,6 +346,12 @@ def prompt_for_ssh(options):
     return True
 
 
+def prompt_for_reboot():
+    print(_("The installation cannot continue and the system will be rebooted"))
+    print(_("Press ENTER to continue"))
+    input()
```

Comment:
Wouldn't this block installation in non-interactive / cmdline mode?

Reply:
Good question, I'll check.

Reply:
So, to check if we are in interactive mode would be non-trivial but doable at this early stage (asking module for kickstart and checking options).
But given sys.exit does not reboot at this stage I think we can just remove the prompt. Then the behaviour would be the same as with kickstart parsing error.